pages/common/speedtest-cli.md

```diff
@@ -0,0 +1,19 @@
+# speedtest-cli
+
+> Command line interface for testing internet bandwidth using speedtest.net.
+
+- Run a speed test:
+
+`speedtest-cli`
+
+- Run a speed test and generate a sharable result picture:
+
+`speedtest-cli --share`
+
+- Print a list of all speedtest.net servers, sorted by distance, to file:
```

Comment:
Let's change this to print to console. I don't see a specific reason to redirect to file.

Reply:
The output is over 7000 lines long (7029 to be specific).

Reply:
Haha, okay then.